client/dead.lua

```diff
@@ -186,8 +194,12 @@ CreateThread(function()
                     end
 
                     if IsControlJustPressed(0, 47) and not emsNotified then
-                        TriggerServerEvent('hospital:server:ambulanceAlert', Lang:t('info.civ_down'))
-                        emsNotified = true
+                        if not MedicCheck() then
+                            lib.notify({ type = 'error', message = Lang:t('error.no_medics') })
+                        else
+                            TriggerServerEvent('hospital:server:ambulanceAlert', Lang:t('info.civ_down'))
+                            emsNotified = true
+                        end
```

Comment:
Consider changing to if MedicCheck() to get rid of the not and doing the second condition first to reduce cognitive complexity.